graph-commons/src/test/scala/ch/datascience/db/DbConfigGenerator.scala

```diff
@@ -29,18 +29,19 @@ import scala.language.postfixOps
 
 private object DbConfigGenerator {
 
-  val dbConfigs: Gen[DBConfig[TestDB]] = for {
-    driver         <- nonEmptyStrings() map (RefType.applyRef[Driver](_).getOrError)
-    url            <- relativePaths() map (RefType.applyRef[Url](_).getOrError)
-    user           <- nonEmptyStrings() map (RefType.applyRef[User](_).getOrError)
-    pass           <- nonEmptyStrings()
-    connectionPool <- positiveInts() map (_.value) map (RefType.applyRef[ConnectionPool](_).getOrError)
-    maxLifetime    <- durations(max = 60 minutes)
-  } yield DBConfig(driver, url, user, pass, connectionPool, maxLifetime)
-
-  implicit class RefinedOps[V](maybeValue: Either[String, V]) {
-    lazy val getOrError: V = maybeValue.fold(s => throw new IllegalArgumentException(s), identity)
-  }
+//  val dbConfigs: Gen[DBConfig[TestDB]] = for {
```

Comment:
Should that go?

Reply:
Yes good catch everything is in the newConfig now as we cannot fake a connection